installer/Product.wxs

```diff
@@ -0,0 +1,507 @@
+				<File Id="ELFIO_WRAPPER.HPP" Name="elfio_wrapper.hpp" Source="..\include\elfio_wrapper.hpp" />
+			</Component>
+			<Component Id="GIT_COMMIT_ID.H" DiskId="1" Guid="0AB71C9D-68E6-4892-A45D-306BA921CC07">
+				<File Id="GIT_COMMIT_ID.H" Name="git_commit_id.h" Source="..\include\git_commit_id.h" />
+			</Component>
+		</ComponentGroup>
+		<ComponentGroup Id="eBPF_Development_inlude_asm" Directory="dir_include_asm">
+			<Component Id="ERRNO.H" DiskId="1" Guid="DF63C9C1-FA9C-44E3-94EB-F77B7072E4BE">
+				<File Id="ERRNO.H" Name="errno.h" Source="..\include\asm\errno.h" />
+			</Component>
+			<Component Id="POSIX_TYPES.H" DiskId="1" Guid="F28F43DC-B53F-4DF9-8E19-F2D7E736E446">
+				<File Id="POSIX_TYPES.H" Name="posix_types.h" Source="..\include\asm\posix_types.h" />
+			</Component>
+			<Component Id="TYPES.H" DiskId="1" Guid="030CB7C2-BD80-4677-A183-890315E75EAC">
+				<File Id="TYPES.H" Name="types.h" Source="..\include\asm\types.h" />
+			</Component>
+		</ComponentGroup>
+		<ComponentGroup Id="eBPF_Development_inlude_bpf" Directory="dir_include_bpf">
+			<Component Id="BPF.H_1" DiskId="1" Guid="A41E9DEB-709E-4DA3-9A24-F5B8C21AF30E">
+				<File Id="BPF.H_1" Name="bpf.h" Source="..\include\bpf\bpf.h" />
+			</Component>
+			<Component Id="BPF_LEGACY.H" DiskId="1" Guid="B3E387BA-03DB-4FE9-A808-B334D48F2CAA">
+				<File Id="BPF_LEGACY.H" Name="bpf_legacy.h" Source="..\include\bpf\bpf_legacy.h" />
+			</Component>
+			<Component Id="HASHMAP.H" DiskId="1" Guid="47680BD2-8931-4098-AFDF-57A5E4688BB5">
+				<File Id="HASHMAP.H" Name="hashmap.h" Source="..\include\bpf\hashmap.h" />
+			</Component>
+			<Component Id="LIBBPF.H" DiskId="1" Guid="2A878010-A1D6-4888-8694-FA4148259282">
+				<File Id="LIBBPF.H" Name="libbpf.h" Source="..\include\bpf\libbpf.h" />
+			</Component>
+			<Component Id="LIBBPF_LEGACY.H" DiskId="1" Guid="62760909-DD06-4711-AFE3-E4FD5A2FE940">
+				<File Id="LIBBPF_LEGACY.H" Name="libbpf_legacy.h" Source="..\include\bpf\libbpf_legacy.h" />
+			</Component>
+		</ComponentGroup>
+		<ComponentGroup Id="eBPF_Development_inlude_kernel" Directory="dir_include_kernel">
+			<Component Id="EBPF_REGISTRY_HELPER.H" DiskId="1" Guid="26717067-176B-4B94-8716-57A675E5F918">
+				<File Id="EBPF_REGISTRY_HELPER.H" Name="ebpf_registry_helper.h" Source="..\include\kernel\ebpf_registry_helper.h" />
+			</Component>
+		</ComponentGroup>
+		<ComponentGroup Id="eBPF_Development_inlude_linux" Directory="dir_include_linux">
+			<Component Id="BPF.H_2" DiskId="1" Guid="ED4F7D03-0334-4FB3-A3AC-1DAD36FF1228">
+				<File Id="BPF.H_2" Name="bpf.h" Source="..\include\linux\bpf.h" />
+			</Component>
+			<Component Id="STDDEF.H" DiskId="1" Guid="41D2F835-B4A7-450F-B064-1280C5D55505">
+				<File Id="STDDEF.H" Name="stddef.h" Source="..\include\linux\stddef.h" />
+			</Component>
+		</ComponentGroup>
+		<ComponentGroup Id="eBPF_Development_inlude_net" Directory="dir_include_net">
```

Comment:
```suggestion
		<ComponentGroup Id="eBPF_Development_include_net" Directory="dir_include_net">
```
make sure you search for all occurrences of "inlude" and fix them

Reply:
fixed